src/backend/debug/debug_crud.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Humanitarian OpenStreetMap Team
```

Comment:
Why is this called debug?

Reply:
It's called debug cause it's only for debugging. Some of the higher level endpoints need to execute several backend support functions. This lets me test them in isolation while debugging their initial implementation. Once the support code is considered stable, each endpoint used for that will get deleted.